exec.go

```diff
@@ -90,7 +87,7 @@ func (db *DB) Insert(table string, data interface{}) error {
 				return err
 			}
 			pk := structInfo.onlyPrimaryKey()
-			if pk != nil && pk.structField.Type.Kind() == reflect.Int64 {
+			if pk != nil && pk.structField.Type.Kind() == reflect.Int64 && row.CanAddr() {
```

Comment:
This should not be necessary, by nature slices are pointers, so they are addressable already. Tests work without this change.